content/_changelogs/9.6.0.md

```diff
@@ -0,0 +1,67 @@
+## 9.6.0
+
+_Released 4/25/2022_
+
+**Experiments:**
+
+- The Sessions beta feature has been enhanced by the new beta feature which
+  enables Multi-Origin support. To reflect the intent of using these experiments
+  together, the `experimentalSessionSupport` configuration option has been
+  replaced by the `experimentalSessionAndOrigin` configuration option. The
+  behaviors and usage of the `cy.session()` command has not been changed. Please
+  update your configuration to use the new `experimentalSessionAndOrigin` option
+  to continue to opt-in to this beta feature.
+
+**Features:**
+
+- We are excited to announce a new beta feature to enable Multi-Origin support.
+  This feature allows a user to visit multiple domains of different origins
+  within a single test. This enhances the current test experience which limits a
+  user to only run assertions against a single origin in a test due to standard
+  web security features of the browser. This experiment was designed to work
+  side-by-wide with the Sessions beta feature. For more details, see [our blog
+  post] (TO DO). Addressed
+  [#17336](https://github.com/cypress-io/cypress/issues/17336) and
+  [#944](https://github.com/cypress-io/cypress/issues/944) -- TO DO.
+  - Added a new configuration option called `experimentalSessionAndOrigin`. This
+    options enabled both Session support and Multi-Origin support.
+  - Added a new Cypress command, called `cy.origin()`, which enables running
+    commands within multiple origins from a single test.
+  - When enabled, cross-origin requests will no longer immediately fail, but
+    instead use the `pageLoadTimeout` configuration as the timeout value. This
+    allows the request to be appropriately handled by the `cy.origin()` command
+    and if it is not handled, it's assumed it was an unexpected request that
+    should fail.
+  - When enabled, tests will no longer wait for the page to load before starting
+    the next tests.
+
```

Comment:
@jennifer-shehane do you feel like the last two bullets are too much? I'm feeling yes. People can read these points in the blog post & the doc page.